### PR TITLE
[fix](regression)Remove useless case which may cause preHeat npe.

### PR DESCRIPTION
**Pull request by Jibing-Li**
Remove useless case which may cause preHeat npe.

### CODE REVIEW

**Comment by Jibing-Li** [comment]
run buildall

File: regression-test/suites/statistics/test_ddl.groovy [hunk 1]
```diff
@@ -44,6 +44,7 @@ suite("test_ddl") {
...
     """
 
+/*
```
Comment (morrySnow) [inline regression-test/suites/statistics/test_ddl.groovy:47]: add some comment to explain why disable this test, and what to do if we want to reopen it. if we do not need it anymore, remove it directly

Comment (Jibing-Li) [reply regression-test/suites/statistics/test_ddl.groovy]: removed

**Comment by Jibing-Li** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Comment by Jibing-Li** [comment]
run p0

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**